regtests/ww3_tp2.22/input/ww3_grid.inp

```diff
@@ -0,0 +1,35 @@
+$ WAVEWATCH III Grid preprocessor input file
+$ ------------------------------------------
+  '2-D PROP. AROUND GLOBE WITH LANGMUIR MIXING PARAMETERIZATION'
+$
+   1.1 0.040 9  24  0.
+$
+   F T T T F F
+  2200.  2200. 2200.  2200.
+$
+$ &PRO2 DTIME = 0. /
```

Comment:
Is there any reason to keep this here commented out, or should we go ahead and clean up this line and line 12?

Reply:
Addressed by: https://github.com/ESCOMP/WW3/pull/15/commits/5b1a73afac53a1cb2d5494caa14ef16039577f98